tests/test_build.py

```diff
@@ -86,19 +87,17 @@ def test_build_book(sphinx_build_factory, file_regression):
         if kernel_name not in thebe_config.prettify():
             raise AssertionError(f"{kernel_name} not in {kernels_expected}")
 
-    # Check a few components that should be true on each page
+    # -- Sidebar --------------------------------------------------------------
+    # navigation entries
     index_html = sphinx_build.html_tree("index.html")
-    sidebar = index_html.find_all(attrs={"class": "bd-sidebar"})[0]
+    sidebar = index_html.find_all(attrs={"class": "bd-sidenav"})[0]
```

Comment:
I think that the important logic we're testing here is in `sidenav` (a child of `sidebar`) and it all behaves the same under `sidenav` for Sphinx 3/4